### ISSUE TITLE
Lua: Fix CMake warning related to policy CMP0025

### ISSUE THREAD

**Issue by jcfr** [SimpleITK/SimpleITK#276]
```
loading initial cache file /home/jcfr/Projects/Slicer-2-build/SimpleITK-build/Lua-prefix/src/Lua-build/CMakeCacheInit.txt

CMake Deprecation Warning at CMakeLists.txt:5 (cmake_policy):
  The OLD behavior for policy CMP0025 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
```

**Comment by blowekamp** [comment]
This has already been addressed in master with 2e5619be9e478c41f795c16a502bb13328864a94.

**Comment by jcfr** [comment]
Makes sense. I reported the issue looking at the current Slicer dashboard .. but I ended up updating the version of SimpleITK in Slicer after the nightly started.
